lib/livebook/session.ex

```diff
@@ -2177,6 +2191,10 @@ defmodule Livebook.Session do
     notify_update(state)
   end
 
+  defp after_operation(state, _prev_state, {:set_notebook_deployment_group, _client_id, _id}) do
+    notify_update(state)
+  end
```

Comment:
We do `notify_update` if the `%Session{}` summary changes, so we don't need it for this event.